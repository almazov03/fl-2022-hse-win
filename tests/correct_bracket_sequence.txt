STARTING_NON_TERMINAL = $S$;
$S$ = { "(" $S$ ")" $S$ }; $S$ = { EPS };