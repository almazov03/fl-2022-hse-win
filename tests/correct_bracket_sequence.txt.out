LexToken(START,'STARTING_NON_TERMINAL',1,0)
LexToken(EQUAL,'=',1,22)
LexToken(NON_TERMINAL,'S',1,24)
LexToken(SEPARATOR,';',1,27)
LexToken(NON_TERMINAL,'S',2,30)
LexToken(EQUAL,'=',2,34)
LexToken(LBR,'{',2,36)
LexToken(TERMINAL,'(',2,38)
LexToken(NON_TERMINAL,'S',2,42)
LexToken(TERMINAL,')',2,46)
LexToken(NON_TERMINAL,'S',2,50)
LexToken(RBR,'}',2,54)
LexToken(SEPARATOR,';',2,55)
LexToken(NON_TERMINAL,'S',2,57)
LexToken(EQUAL,'=',2,61)
LexToken(LBR,'{',2,63)
LexToken(EMPTY,'EPS',2,65)
LexToken(RBR,'}',2,69)
LexToken(SEPARATOR,';',2,70)
